src/main/scala/is/hail/variant/Genotype.scala

```diff
@@ -328,6 +335,7 @@ object Genotype {
     val m = Int.MaxValue / (nAlleles + 1)
     val nGenotypes = triangle(nAlleles)
     val gg = for (gt: Option[Int] <- Gen.option(Gen.choose(0, nGenotypes - 1));
```

Comment:
Move this into a Call.gen: Gen[Call].  That'll make it easier to add phase and multi-ploidy support.